common/test/app/common/pages_migrations_test.cljc

```diff
@@ -71,7 +71,7 @@
                  :components {}
                  :version 7}
 
-        expct   (-> data
+        expect   (-> data
```

Comment:
I'm not sure how important whitespace is to clojure...

Reply:
it is ok ;)